Framework/PythonInterface/plugins/algorithms/RebinRagged.py

```diff
@@ -41,7 +41,14 @@ def PyInit(self):
         self.declareProperty("PreserveEvents", True, "False converts event workspaces to histograms")
 
     def validateInputs(self):
+        errors = {}
+
         inputWS = self.getProperty("InputWorkspace").value
+
+        if not isinstance(inputWS, MatrixWorkspace):
+            errors['InputWorkspace'] = "The InputWorkspace must be a MatrixWorkspace."
+            return errors
```

Comment:
Again, is the early return necessary?

Reply:
I have shuffled the validation around a bit so that the early return is no longer used